.github/workflows/azure_automation_test.yml

```diff
@@ -158,14 +193,16 @@ jobs:
   #
   # Test Run
   #
-
+  
+  # Here we specify we need `create-bee-workflow`, but `if: always()` ensures this is run even if that step is skipped
   leo-automation-test-job:
     runs-on: ubuntu-latest
     needs:
       - init-github-context
-      - create-bee-workflow
       - params-gen
       - attach-billing-project-to-landing-zone-workflow
+      - create-bee-workflow
+    if: always()
```

Comment:
if we have the `always()` here, is the `needs` section necessary, since it'll run no matter what?